sdk/core/core-amqp/src/util/checkNetworkConnection.ts

```diff
@@ -10,25 +10,25 @@ import { logger } from "../log";
  * @internal
  */
 export function checkNetworkConnection(host: string): Promise<boolean> {
-  return new Promise((res) => {
+  return new Promise((resolve) => {
     logger.verbose("Calling dns.resolve to determine network connection status.");
-    resolve(host, function(err: any): void {
+    dnsResolve(host, function(err: any): void {
       if (err) {
         logger.verbose(
           "Error thrown from dns.resolve in network connection check: '%s', %O",
           err.code || err.name,
           err
         );
         // List of possible DNS error codes: https://nodejs.org/dist/latest-v12.x/docs/api/dns.html#dns_error_codes
-        // Only when dns.resolve returns an error we expect to see when the network is down, resolve as 'false'.
+        // Only when dns.dnsResolve returns an error we expect to see when the network is down, dnsResolve as 'false'.
```

Comment:
https://github.com/Azure/azure-sdk-for-js/pull/8680/#discussion_r420936577 applies here as well

Either refer to the function as `dns.resolve` or `dnsResolve`

Reply:
Thank you, I missed this